README.md

```diff
@@ -1 +1,50 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project containing several file such as: 
+1. index.html
+2. style.css
+3. linters.yml
+
+    This project was built to display a portfolio to display two parts the toolbar which contains My logo and a SVG file. The second part was to build a headline section right after the header which displayed a Hello message, a description of the portfolio owner and the links in which he can be contacted.
+
+
+## Built With
+
+- HTML, CSS, Flexbox
+- Visual Studio Code
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+Dear User, in order to be able to open this project in your device kindly click on the green code button and select between the given options: 
+1. Open in Github Desktop.
+2. Download the ZIP file and open it in VS Code.
+3. Copy the SSH link and clone it in your terminal or powershell.
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+    You only need a good computer with a browser and VS Code.
+
+
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] Kindly replace Author1 with your name as it will be clear who is the author and looks professional.